pages/docs/reference/whatsnew14.md

```diff
@@ -578,31 +578,15 @@ Learn [how to add dependencies](native/cocoapods.html).
 
 ## Standard library
 
-### Extending the common library
+### Common exception processing API
 
-_Common_ library is the part of the standard library that can be used in common code shared between multiple platforms.
-In 1.4, we continue extending the common library and adding or moving missing functionality to it:
-
-- `appendLine()` function that appends a given string and terminates it with `\n`:
```

Comment:
Now `appendLine` seems missing. Also there was a big change of `StringBuilder` API in common, js, and native platforms: https://youtrack.jetbrains.com/issue/KT-33069, some new extensions were introduced in JVM as well. @qurbonzoda can provide the details.

Reply:
https://github.com/JetBrains/kotlin-web-site/pull/1787#issuecomment-666515415